SparkPost/2web2ui
# SA-1982: Add segment tracking on creating scheduled report

### What Changed

- Added Segment tracking for Creating a scheduled report

### How To Test

- Create a scheduled report; `/signals/schedule/bounce`
- Go to https://app.segment.com/sparkpost-daniel-chalef/sources/webui_dev/debugger
- See if the event showed up.

### To Do

- [ ] MixPanel Check?
